### ISSUE TITLE
writerpi: not found

### ISSUE THREAD

**Issue by dwehrmann** [picnoir/ultimate-writer#11]
First of all, great project. Thanks for putting this all together!

Getting through the installation with some warnings.

Starting the service works, but `status` says:

`Mar 17 02:35:39 raspiwh systemd[1]: Started "Ultimate writer e-ink screen driver".
Mar 17 02:35:40 raspiwh start-ultimatewriter[8279]: /usr/local/bin/start-ultimatewriter: 3: /usr/local/bin/writerpi: not found
Mar 17 02:35:40 raspiwh systemd[1]: ultimatewriter.service: Main process exited, code=exited, status=127/n/a
Mar 17 02:35:40 raspiwh systemd[1]: ultimatewriter.service: Failed with result 'exit-code'.`

Which does not look good. My waveshare 7.5V2 flashes a couple of times and a lot of pixels appear, however smaller than the full screen.
I would really like to get this working. Any advice is highly appreciated.

**Comment by za3k** [comment]
Sounds like you might be using the V1 software. Double-check?

**Comment by dwehrmann** [comment]
Yes, I suppose so. I downloaded the 0.1 release as I couldn't make the latest version. 
`make` threw at me:

```
st build options:
CFLAGS  =  -DVERSION="0.1" -D_XOPEN_SOURCE=600 -fcommon
LDFLAGS = -L -lm -lutil -lpthread -lbcm2835
CC      = c99
c99  -DVERSION=\"0.1\" -D_XOPEN_SOURCE=600 -fcommon -DSCREEN_75_V2 -c st.c
In file included from st.c:35:
screen.h:68:2: error: #endif without #if
   68 | #endif
      |  ^~~~~
st.c: In function ‘twrite’:
st.c:814:30: warning: passing argument 1 of ‘utf8decode’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
  814 |    charsize = utf8decode(buf + n, &u, buflen - n);
      |                          ~~~~^~~
st.c:265:18: note: expected ‘char *’ but argument is of type ‘const char *’
  265 | utf8decode(char *c, Rune *u, size_t clen)
      |            ~~~~~~^
make: *** [Makefile:26: st.o] Error 1
```

**Comment by vcruzdesigns** [comment]
Hi @dwehrmann, were you able to get it to work? I've run into the same error.

**Comment by dwehrmann** [comment]
@vcruzdesigns No, I eventually switched to https://github.com/joukos/PaperTTY which worked great for what I was building.
